### PR TITLE
VMS: simplify config targets

### PR DESCRIPTION
**Pull request by levitte**
All VMS config targets were literally copies of each other, only
differing in what argument the parameter seeking function vms_info()
received (the pointer size).

This could be hugely simplified by letting vms_info() detect what
pointer size was desired from the desired config target name instead.

### CODE REVIEW

**Review by dot-asm — APPROVED** [review]

File: Configurations/10-main.conf [hunk 1]
```diff
@@ -1834,132 +1840,72 @@ my %targets = (
...
                                    release => "/NODEBUG/NOTRACEBACK"),
         lib_cflags       => add("/NAMES=(AS_IS,SHORTENED)/EXTERN_MODEL=STRICT_REFDEF"),
         # no_inst_lib_cflags is used instead of lib_cflags by descrip.mms.tmpl
         # for object files belonging to selected internal libraries
         no_inst_lib_cflags => "",
+        ex_libs          => add(sub { return vms_info()->{zlib} || (); }),
         shared_target    => "vms-shared",
         dso_scheme       => "vms",
         thread_scheme    => "pthreads",
+        bn_ops           => "SIXTY_FOUR_BIT RC4_INT",
```
[the remark this answers is not in the document]
Comment (levitte) [reply Configurations/10-main.conf]: Well, VAX is gone...  but you make a good point, it shouldn't be a default like this...

**Comment by levitte** [comment]
Side note got me to do a small change.  Reconfirm?

**Comment by dot-asm** [comment]
May I ask for another modification then? Could you add a comment about what does 64=ARGV achieve? Or maybe what trouble does it prevent? Doesn't have to be long...

**Comment by levitte** [comment]
Certainly!

**Comment by levitte** [comment]
Merged. ab90718953eabbd8fa519895387d065c94418f5c

**Pull request closed without merging**